crates/db/src/commands.rs

```diff
@@ -44,8 +44,17 @@ pub async fn db_get_native_balance(
 #[tauri::command]
 pub async fn db_get_erc721_tokens(
     chain_id: u32,
-    address: Address,
+    owner: Address,
     db: tauri::State<'_, DB>,
 ) -> Result<Vec<Erc721Token>> {
-    db.get_erc721_tokens(chain_id, address).await
+    db.get_erc721_tokens(chain_id, owner).await
+}
+
+#[tauri::command]
+pub async fn db_get_erc721_collections(
```

Comment:
what is the need for this command?

Reply:
The idea was for the frontend to store collection data and avoid making multiple database reads for tokens from the same collection. This approach also helps in avoiding the need to create a specific type just to return the token and collection in a single request. Additionally, it could enable different displays on the frontend, such as having collections and then entering to view the tokens, or something similar